core/src/test/java/org/elasticsearch/common/xcontent/ObjectParserTests.java

```diff
@@ -34,6 +35,8 @@
 
 public class ObjectParserTests extends ESTestCase {
 
+    private static ParseFieldMatcherSupplier strictParsing = () -> ParseFieldMatcher.STRICT;
```

Comment:
also if it's a constant it should probably be STRICT_PARSING instead? but why do we need this constant again? :)